sdk/src/main/scala/com/horizen/account/utils/WellKnownAddresses.scala

```diff
@@ -0,0 +1,14 @@
+package com.horizen.account.utils
+
+import com.horizen.utils.BytesUtils
+
+object WellKnownAddresses {
+
+  // fake smart contract address
+  val WITHDRAWAL_REQ_SMART_CONTRACT_ADDRESS_BYTES: Array[Byte] = BytesUtils.fromHexString("0000000000000000000011111111111111111111")
+  val FORGER_STAKE_SMART_CONTRACT_ADDRESS_BYTES: Array[Byte] = BytesUtils.fromHexString("0000000000000000000022222222222222222222")
+
+  // used to burn coins
+  val DEAD_ADDRESS_BYTES: Array[Byte] = BytesUtils.fromHexString("000000000000000000000000000000000000dead")
```

Comment:
The null address shall be 0x000…000, like one of the most commonly used null addresses in Ethereum https://etherscan.io/address/0x0000000000000000000000000000000000000000.

Reply:
Done